oca_port/utils/git.py

```diff
@@ -13,6 +13,21 @@
 from .misc import bcolors as bc, pr_ref_from_url
 
 PO_FILE_REGEX = re.compile(r".*i18n/.+\.pot?$")
+TRANSLATION_SUMMARY = [
+    "Added translation using Weblate",
+    "Translated using Weblate",
+]
+SQUASHABLE_SUMMARY = [
+    "Update translation files",
+]
+SQUASHABLE_AUTHOR_EMAIL = [
+    "transbot@odoo-community.org",
+    "noreply@weblate.org",
+    "oca-git-bot@odoo-community.org",
+    "oca+oca-travis@odoo-community.org",
+    "oca-ci@odoo-community.org",
+    "shopinvader-git-bot@shopinvader.com",
+]
```

Comment:
These values are also declared in `port_addon_pr.py` module.
I would rename `SQUASHABLE_AUTHOR_EMAIL` to `AUTHOR_EMAILS`, then use them in your code below and import them in `port_addon_pr.py` in replacement of `AUTHOR_EMAILS_TO_SKIP` and `SUMMARY_TERMS_TO_SKIP`